### PR TITLE
Migrated from fluentUI Northstar library package to React-components library.

### PR DESCRIPTION
**Pull request by Pawan-MSFT**

### CODE REVIEW

File: samples/meetings-token-app/nodejs/client/src/components/UserList.js [hunk 1]
```diff
@@ -13,11 +12,12 @@ class TokenActionButtons extends Component {
     }
 
     render() {
+        const listItems = this.props.items.map((myList)=>{
```
Comment (ChetanSharma-msft) [inline samples/meetings-token-app/nodejs/client/src/components/UserList.js:15]: Let's rename variables to use meaningful name.

File: samples/tab-channel-group-quickstart/js/src/components/App.js [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import React from 'react';
 import './App.css';
 import * as microsoftTeams from "@microsoft/teams-js";
-import { BrowserRouter as Router, Route } from "react-router-dom";
+import { BrowserRouter,Route ,Routes} from 'react-router-dom';
```
Comment (ChetanSharma-msft) [inline samples/tab-channel-group-quickstart/js/src/components/App.js:7]: Add spacing after comma

File: samples/tab-channel-group-quickstart/js/src/index.js [hunk 3]
```diff
@@ -5,10 +5,10 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './components/App';
-import { Provider, themes } from '@fluentui/react-northstar' //https://fluentsite.z22.web.core.windows.net/quick-start
+import { FluentProvider, teamsDarkTheme } from '@fluentui/react-components' //https://fluentsite.z22.web.core.windows.net/quick-start
```
Comment (ChetanSharma-msft) [inline samples/tab-channel-group-quickstart/js/src/index.js:8]: Let's remove or add relevant URL in place of below:
//https://fluentsite.z22.web.core.windows.net/quick-start

File: samples/tab-channel-group-quickstart/ts/src/components/App.tsx [hunk 4]
```diff
@@ -3,8 +3,7 @@
 
 import './App.css';
 import * as microsoftTeams from "@microsoft/teams-js";
-import { BrowserRouter as Router, Route } from "react-router-dom";
-
+import { BrowserRouter,Route ,Routes} from 'react-router-dom';
```
Comment (ChetanSharma-msft) [inline samples/tab-channel-group-quickstart/ts/src/components/App.tsx:6]: Add proper spacing in between comma like below 
{ BrowserRouter, Route, Routes }

File: samples/tab-personal-quickstart/js/src/index.js [hunk 5]
```diff
@@ -5,10 +5,10 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './components/App';
-import { Provider, themes } from '@fluentui/react-northstar' //https://fluentsite.z22.web.core.windows.net/quick-start
+import { FluentProvider,   teamsLightTheme } from '@fluentui/react-components' //https://fluentsite.z22.web.core.windows.net/quick-start
```
Comment (ChetanSharma-msft) [inline samples/tab-personal-quickstart/js/src/index.js:8]: Let's remove below or add relevant URL:
//https://fluentsite.z22.web.core.windows.net/quick-start

**Review by ChetanSharma-msft — APPROVED** [review]
Added my review comments.

**Comment by ChetanSharma-msft** [comment]
**Executed build pipeline:** https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/3604680181
**Result:**

**Review by Wajeed-msft — APPROVED** [review]

File: samples/tab-channel-group-quickstart/js/src/components/App.js (hunk 2 above)
Comment (Pawan-MSFT) [reply samples/tab-channel-group-quickstart/js/src/components/App.js:7]: Fixed

**Pull request merged**